test/over_react/util/dom_util_test.dart

```diff
@@ -301,23 +301,29 @@ main() {
       });
 
       group('for an `<input>` of type:', () {
-        void sharedInputGetSelectionStartTest(String type) {
+        void sharedInputGetSelectionStartTest(String type, {bool shouldReturnNull: false}) {
           renderedInstance = renderAttachedToDocument((Dom.input()
             ..defaultValue = testValue
             ..type = type
           )());
           inputElement = findDomNode(renderedInstance);
+          setSelectionRange(inputElement, testValue.length, testValue.length);
+
           var selectionStart = getSelectionStart(inputElement);
 
-          expect(selectionStart, testValue.length);
+          expect(selectionStart, shouldReturnNull ? isNull : testValue.length);
         }
 
         for (var type in inputTypesWithSelectionRangeSupport) {
           if (type == 'email' || type == 'number') {
             // See: https://bugs.chromium.org/p/chromium/issues/detail?id=324360
+            test('$type, returning null in Google Chrome (or Dartium/content_shell) or Firefox', () {
+              sharedInputGetSelectionStartTest(type, shouldReturnNull: true);
+            }, testOn: 'blink || firefox');
+
             test(type, () {
-              sharedInputGetSelectionStartTest(type);
-            }, testOn: 'js && !chrome');
+              sharedInputGetSelectionStartTest(type, shouldReturnNull: false);
+            }, testOn: '!(blink || firefox)');
```

Comment:
Ugh, it's because coverage doesn't include the platform: https://github.com/Workiva/dart_dev/issues/200. I'll apply a fix for that